docs/source/crawler/basics.rst

```diff
@@ -20,12 +20,47 @@ a modified dictionary as its output once.
 >>> next(add1_node({'input': 1}))
 {'input': 2}
 
+.. Generators::
+    Generators may be one of many things including, but not limited to: nodes,
+    independent functions, or as methods of a class. Generators may yield a
```

Comment:
@yarikoptic should I be giving examples or types of generators under this node section?